### DISCUSSION TITLE
Can this crate be used asynchronously?

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [rusqlite/rusqlite#1364] [Questions] [answered]
What I mean by that is if it's possible run functions like `execute` async

**Comment by gwenn** [comment, marked as answer]
No.
[sqlx](https://docs.rs/sqlx/latest/sqlx/) is async.
But [this](https://github.com/diesel-rs/metrics/) page suggests that there is an impact on perf.

**Reply by MordechaiHadad** [reply]
thank you!
